definitions/variable/technology/power-plant.yaml

```diff
@@ -194,58 +185,48 @@
     description: This is the upper bound that energy storage system could charge energy
       at time "t".
     unit:
-    - MWh
     - GWh
 - Initial Storage Level|Electricity|Energy Storage System:
     description: Initial energy stored in the energy storage system unit at first
       load level. (ESS that includes hydro, battery, etc.)
     unit:
-    - MWh
     - GWh
 - Maximum Charge|Electricity|Energy Storage System:
     description: Maximum charge when storing energy the energy storage system unit.
     unit:
-    - MW
     - GW
 - Maximum Discharge|Electricity|Energy Storage System:
     description: Maximum discharge when consuming energy from the energy storage system
       unit.
     unit:
-    - MW
     - GW
 - Maximum Storage|Electricity|Energy Storage System:
     description: Maximum energy that can be stored by the energy storage systems.
     unit:
-    - MWh
     - GWh
 - Maximum Storage|Electricity|Energy Storage System|Lithium-Ion:
     description: Maximum energy that can be stored by the Lithium Iron Batteries systems.
     unit:
-    - MWh
     - GWh
 - Maximum Storage|Electricity|Energy Storage System|Redox Flow:
     description: Maximum energy that can be stored by the Redox Flow Batteries systems.
     unit:
-    - MWh
     - GWh
 - Maximum Storage|Electricity|Energy Storage System|Compressed Air:
     description: Maximum energy that can be stored by the Compressed air energy storages
       systems.
     unit:
-    - MWh
     - GWh
 - Minimum Storage|Electricity|Energy Storage System:
     description: Minimum energy that can be stored by the energy storage system unit.
     unit:
-    - MWh
     - GWh
 - Spillage|Electricity|Energy Storage System:
     description: Energy spillage that could not be stored by an energy storage system
       (ESS). Several energy models include spillage variables related to renewable
       energy production; however, another way is to include a spillage variable to
       the ESS unit since it is the last sink of the power network.
     unit:
-    - MWh
     - GWh
 - Storage Type|Electricity|Energy Storage System:
```

Comment:
If 'type' is related only to seasonality, why not call this variable Seasonality instead of Storage Type?

Reply:
Got it!